ivy/data_classes/array/array.py

```diff
@@ -363,8 +364,7 @@ def __repr__(self):
                 self._post_repr = ", dev={})".format(self._dev_str)
             else:
                 self._post_repr = ")"
-        sig_fig = ivy.array_significant_figures
-        dec_vals = ivy.array_decimal_values
+        dec_vals = ivy.array_decimal_values()
```

Comment:
please revert the change to `dec_vals` as `ivy.array_decimal_values` isn't a function